src/core/src/pass/serialize.cpp

```diff
@@ -953,10 +1034,12 @@ void ngfunction_2_ir(pugi::xml_node& netXml,
         }
 
         // fill <data> general attributes
-        auto_pad_resolving(node);  // Backward compatibility: clear padding values for nodes with auto_pad
+        auto old_pads =
+            auto_pad_resolving(node);  // Backward compatibility: clear padding values for nodes with auto_pad
```

Comment:
can we clone node and perform all changes on cloned node? Original one will be kept as is to prevent issues in multi-threading environment